prov/shm/src/smr_attr.c

```diff
@@ -68,7 +70,7 @@ struct fi_domain_attr smr_domain_attr = {
 	.data_progress = FI_PROGRESS_MANUAL,
 	.resource_mgmt = FI_RM_ENABLED,
 	.av_type = FI_AV_UNSPEC,
-	.mr_mode = FI_MR_VIRT_ADDR,
+	.mr_mode = OFI_MR_BASIC_MAP | FI_MR_BASIC,
```

Comment:
This doesn't look right.  Shm doesn't require memory registration for local buffers -- only for RMA target.  The app doesn't need to register send or receive buffers, or buffers that are used on the initiator side of RMA/atomic operations.